### PR TITLE
StorageInterface

### PR DESCRIPTION
**Pull request by Mejiao99**

### CODE REVIEW

File: src/main/java/de/maxhenkel/camera/StorageInterface.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package de.maxhenkel.camera;
+
+interface StorageInterface {
+    void saveImage();
```
Comment (jmena) [inline src/main/java/de/maxhenkel/camera/StorageInterface.java]: Receive `String id, ByteBuffer data`

File: src/main/java/de/maxhenkel/camera/StorageInterface.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+    void loadImage();
```
Comment (jmena) [inline src/main/java/de/maxhenkel/camera/StorageInterface.java]: Receives an `String id` returns an `Optional<ByteBuffer>`

File: src/main/java/de/maxhenkel/camera/StorageInterface.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package de.maxhenkel.camera;
+
```
Comment (jmena) [inline src/main/java/de/maxhenkel/camera/StorageInterface.java:2]: Are there other interfaces in this project? What’s the naming convention?

Comment (Mejiao99) [reply src/main/java/de/maxhenkel/camera/StorageInterface.java:2]: None

**Pull request closed without merging**